### PR TITLE
Show content of map

### LINKED ISSUE #3490: EntryLogger: useless logging of ledgersMap
```
DEBUG org.apache.bookkeeper.bookie.EntryLogger - Retrieved entry log meta data entryLogId: 85, meta: { totalSize = 969569129, remainingSize = 969569129, ledgersMap = org.apache.bookkeeper.util.collections.ConcurrentLongLongHashMap@58a3372e }
```

expected to see content of the map

### PR DESCRIPTION
**Pull request by gaozhangmin**
Fixes #3490
Change `ToString` method, print the content of map.

### CODE REVIEW

**Comment by gaozhangmin** [comment]
@dlg99 PTAL

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLogMetadata.java [hunk 1]
```diff
@@ -105,7 +106,7 @@ public void removeLedgerIf(LongPredicate predicate) {
     public String toString() {
         StringBuilder sb = new StringBuilder();
         sb.append("{ totalSize = ").append(totalSize).append(", remainingSize = ").append(remainingSize)
-                .append(", ledgersMap = ").append(ledgersMap).append(" }");
+                .append(", ledgersMap = ").append(StringUtils.join(ledgersMap)).append(" }");
```
Comment (hangc0276) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLogMetadata.java:109]: We have overridden the `toString()` method for ledgersMap. Do we need to add `join` for this map?
https://github.com/apache/bookkeeper/blob/cbb74eab4f1f464c0e4760acf9304af2289acfa1/bookkeeper-server/src/main/java/org/apache/bookkeeper/util/collections/ConcurrentLongLongHashMap.java#L914-L927

Comment (gaozhangmin) [reply bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLogMetadata.java]: Thx for pointing it out, fixed. @hangc0276

**Review by hangc0276 — APPROVED** [review]

**Review by wenbingshen — APPROVED** [review]
lgtm

**Review by HQebupt — APPROVED** [review]

**Review by StevenLuMT — CHANGES_REQUESTED** [review]
can you add a testcase to check the right result?

File: bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/DefaultEntryLogTest.java [hunk 2]
```diff
@@ -190,6 +190,9 @@ public void testCorruptEntryLog() throws Exception {
         entryLogger = new DefaultEntryLogger(conf, dirsMgr);
 
         EntryLogMetadata meta = entryLogger.getEntryLogMetadata(0L);
+        String metaString = meta.toString();
+        assertEquals(metaString,
+                "{totalSize = 60, remainingSize = 60, ledgersMap = ConcurrentLongLongHashMap{1 => 30, 3 => 30}}");
```
Comment (StevenLuMT) [inline bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/DefaultEntryLogTest.java:195]: good job

**Review by StevenLuMT — APPROVED** [review]

**Comment by gaozhangmin** [comment]
rerun failure checks

**Comment by gaozhangmin** [comment]
rerun failure checks

**Comment by zymap** [comment]
Branch-4.15 tests failed. I found this PR depends on PR #3256. Revert it from branch-4.15.

**Pull request merged**